include/cmock/cmock-function-class-mockers.h.pump

```diff
@@ -104,6 +95,12 @@ GMOCK_RESULT_(, F) n($declare_args) { \
         return mock->n($call_args); \
     } \
 	\
+    if (__cmock_real_##c##_##n == NULL) { \
+        std::ostringstream msg; \
```

Comment:
What do you think about changing the message to _Error: Function #n not found. Neither mock nor real function is present._

Reply:
Done